src/wallet/mod.rs

```diff
@@ -5414,4 +5417,40 @@ pub(crate) mod test {
         // ...and checking that everything is fine
         assert_fee_rate!(psbt, details.fee.unwrap_or(0), fee_rate);
     }
+
+    #[cfg(feature = "test-hardware-signer")]
+    #[test]
+    fn test_create_signer() {
+        use hwi::{types, HWIClient};
```

Comment:
nit: instead of including `types` and always using `types::HWIChain`, just include `HWIChain`